cpp/src/slice2cs/Gen.cpp

```diff
@@ -29,33 +29,10 @@ namespace
 {
 
 string
-sliceModeToIceMode(Operation::Mode opMode, string ns)
+sliceModeToIdempotent(Operation::Mode opMode)
```

Comment:
maybe rename this as `sliceModeIsIdempotent`

Reply:
Replaced by isIdempotent, even cleaner!